### ISSUE TITLE
Cocoapods integration

### ISSUE THREAD

**Issue by ilkerc** [KittyMac/Sextant#2]
It would be great to utilize sextant as a cocoapods dependency inside a cocoapods framework. I did some search and looks like spm dependency never got implemented on cocoapods. To do that we should also add cocoapods support to Chronometer, Spanker and Hitch.

Also; I still couldn't find a solution for running ios apps in simulator on an m1 based mac, without enabling rosetta in xCode (not a good idea). `Could not find module ‘Sextant’ for target ‘x86_64-apple-ios-simulator’;`

What do you think @KittyMac ?

**Comment by KittyMac** [comment]
@ilkerc I personally don't have much interest in supporting Cocoapods.  If you'd like to submit PRs to add support for it I will review them for inclusion.

As far as using Sextant in iOS simulator on an M1, I was unable to reproduce your issue.  I created new app project in Xcode on iMac M1, linked to Sextant using the SPM integration in Xcode, added a simple func to use Sextant and ran it on the simulator no problem.  I am not certain if I "enabled rosetta in XCode" in the past, and a quick google did not lead to an easy solution to tell me how to remove it if I had enabled it.
